index.md

```diff
@@ -2,27 +2,48 @@
 sd_hide_title: true
 ---
 
-# The PyMC project
-PyMC is a community driven project with the goal of making Bayesian modeling
-and probabilistic programming intuitive and performant.
+## The PyMC library
```

Comment:
I would keep the project section and the announcements first so that both but especially the announcements are visible without needing to scroll down. The PyMC library is the main thing we work on and it is around the PyMC library that the PyMC community is built but the library already has the docs.pymc.io website.

IMO what should be available here at a glance are the announcements, not the interactive demo whereas the opposite happens back at docs.pymc.io. We can also have the demo in both places, but I think it is perfectly fine if it is a bit hidden here.

Reply:
Hm, so what is the main entry point for new users?

Reply:
I would have thought pymc.io for people interested in the pymc community and docs.pymc.io for people interested in the library. But maybe it is better to have a single entry point here at pymc.io and remove things from the docs.pymc.io homepage.

Reply:
Checking what do other projects do.

Numpy: https://numpy.org/ as unique entry point and https://numpy.org/doc/stable/ as documentation homepage (our docs.pymc.io)
xarray: same structure https://xarray.dev/ and https://docs.xarray.dev/en/stable/
bokeh: very similar too http://bokeh.org/ and http://docs.bokeh.org/en/latest/
matplotlib: also similar https://matplotlib.org/ and https://matplotlib.org/stable/

Reply:
@OriolAbril For what it's worth, I get confused navigating around PyMC website. Confused about the URLs and finding stuff.
It's so much easier with scikit-learn:  https://scikit-learn.org/dev/index.html

Reply:
there is more variation on the documentation homepage (with the tonic of being much less flashy and appealing than the general homepage) we can look into that later. But for the general homepage the structure is quite similar in all projects. The main elements are:

* name+logo with a link/button to either docs or getting started page
* announcements/news: a banner in numpy a bit lower as one out of two columns in matplotlib
* key features
* try the library
* ecosystem
* case studies/examples/common scenarios
* support

so it is probably better to keep this order, maybe extend the contents of the page a bit more too.

Reply:
I think we could have

* pymc library key features section (already there) and has the link to the docs, we could optionally format it as a button too
* try the library (already there too)
* announcements (also there)
* (ecosystem, it could be interesting to highlight some libraries that build on top of pymc like bambi or exoplanet and also keep the separated page with all the libraries that depend on it, not sure if we want to add this already or wait a bit)
* case studies: we could 
* project wide activities (already there)
* support

how does this sound @twiecki @reshamas?

> I get confused navigating around PyMC website. Confused about the URLs and finding stuff.

what are the main sources of confusion? Does this also happen with any of the libraries I linked above, they all have a very similar structure in their websites

Reply:
FWIW the structure of pymc.io and docs.pymc.io makes sense to me but I know I also am not the target user

Reply:
> FWIW the structure of pymc.io and docs.pymc.io makes sense to me but I know I also am not the target user

The current one, this proposal or the fact that they are separated websites?